packages/extension-base/src/services/subscan-service/subscan-chain-map.ts

```diff
@@ -57,7 +57,12 @@ const SUBSCAN_CHAIN_MAP: Record<string, string> = {
   bajun: 'bajun',
   snow: 'snow',
   kilt_peregrine: 'kilt-testnet',
-  polymesh: 'polymesh'
+  polymesh: 'polymesh',
```

Comment:
Please note, this information will be replaced with `chainInfo.extraInfo?.subscanSlug`